Dockerfile

```diff
@@ -11,9 +11,7 @@ LABEL RUN="docker run -d -p 9000:9000 -v <silo db/log location>:/usr/silo silo"
 WORKDIR /silo
 COPY silo /silo
 
-# Get root certs, if certs arg is present
-RUN if [ "x$CERT_PATH" != "x" ] ; then yum install -y wget && update-ca-trust enable && wget $CERT_PATH -r -A *.cer -nd -nv -P /etc/pki/ca-trust/source/anchors/ && update-ca-trust extract; fi \
```

Comment:
If we take out `CERT_PATH` we will have no way to inject root trust in our build for self-signed certificates and certificates for air-gapped networks. Did you have an alternative way you wanted to accomplish this going forward?  I suppose a base image would allow us to do that just as well. I think I talked myself into this. :-)

Reply:
Yeah, I moved to a base image for this as I had to use one when building on said network anyway.